docs/reference/commandline/swarm_init.md

```diff
@@ -128,6 +130,14 @@ management traffic of the cluster.
 If unspecified, Docker will use the same IP address or interface that is used for the
 advertise address.
 
+### `--default-addr-pool`
+This flag specifies default subnet pools for global scope networks.
+Format example is `--default-addr-pool 30.30.0.0/16 --default-addr-pool 40.40.0.0/16`
```

Comment:
I'll add a "docs-revisit" label; we'll probably want some more information about this feature (more details how it will be used by SwarmKit etc)